glusterd2/store/config.go

```diff
@@ -20,6 +20,12 @@ const (
 	etcdCURLsOpt     = "etcdcurls"
 	etcdPURLsOpt     = "etcdpurls"
 	etcdLogFileOpt   = "etcdlogfile"
+	useTLS           = "usetls"
+	certFile         = "cert-file"
+	keyFile          = "key-file"
+	caFile           = "caFile"
+	clntCertFile     = "client-cert-file"
```

Comment:
It would be preferrable to add this to `glusterd2/config.go`. I'd want to use the same options for tls certs for other servers.

Reply:
done. will update the PR accordingly.

Reply:
Moving the constants to `glusterd2/config.go` causes cyclic dependencies as below

```
package github.com/gluster/glusterd2/glusterd2
	imports github.com/gluster/glusterd2/glusterd2/daemon
	imports github.com/gluster/glusterd2/glusterd2/events
	imports github.com/gluster/glusterd2/glusterd2/store
	imports github.com/gluster/glusterd2/glusterd2
```

How to go about this?

Reply:
I feel rather we can have a `commons` package with all the common stuff defined which could be used across modules like `glusterd2`, `glustercli` etc.

@kshlm thoughts??